hnn_core/hnn_io.py

```diff
@@ -196,7 +209,7 @@ def _read_connectivity(net, conns_data):
     # Overwrite drive connections
     net.connectivity = list()
 
-    for i, conn_data in enumerate(conns_data):
+    for _i, conn_data in enumerate(conns_data):
```

Comment:
Why the `enumerate` here? It looks like `i` is unused.

Reply:
I'm not sure. I just left it in there. I suppose someone had added it for debugging purposes. I just added the underscore for for my linter to stop yelling at me. 😅  I suppose no one will miss it if I remove it.